### ISSUE TITLE
Viewing Implementation Report freezes GUI

### ISSUE THREAD

**Issue by pjotrek-b** [MediaArea/MediaConch#132]
There are several cases within the MediaConch GUI, where the application seems frozen, although it's just working in the background.

I've noticed that also trying to change the verbosity using the drop-down in the report HTML viewer causes the same effect (because it has to render it again, I presume).

It's okay to wait, but **would it be possible to have any progress indicator?**
Just to show the user that the requested action might take "a bit longer" to finish?

Without this "wait-indicator", it's currently a bit frustrating to use the application, because it very often is busy - without telling you anything... :disappointed:

**Comment by pjotrek-b** [comment]
Very likely related to, or duplicate of #130.

**Comment by JeromeMartinez** [comment]
We are technically a bit blocked by our XML/XSLT library, which takes a lot of time and without progress indicator.
We are working on improving performance (less data in the input XML, "micromediatrace" optimized for automatic analysis rather than human readable) of the XML part, as well as providing a better indicator during processing of the file.

**Comment by dericed** [comment]
Can we proceed with micro-mediatrace? That gave a big xsltproc speedboost in my tests.

**Comment by JeromeMartinez** [comment]
> Can we proceed with micro-mediatrace? 

we are working on a complete rewrite of the trace part, before adding more trace "formats". 
The code is too complex right now, we have too many bugs with it, it is better to refactor now than dealing with another output format and new bugs.

By curiosity, a number for the big speedboost ratio?
